### PR TITLE
Minor documentation clean up

### LINKED ISSUE #1221: Minor documentation clean up
To match other NEST repos:

- Fix capitalization of function titles (sentence case)
- Remove unnecessary quotes around terms

### PR DESCRIPTION
**Pull request by edelarua**
# Pull Request

Fixes #1221

### CODE REVIEW

File: R/g_forest.R [hunk 1]
```diff
@@ -473,6 +471,8 @@ g_forest <- function(tbl,
 #'
 #' draw_grob(p)
 #' }
+#'
+#' @noRd
```
Comment (Melkiades) [inline R/g_forest.R:475]: should we add the keyword internal?

File: R/formatting_functions.R [hunk 2]
```diff
@@ -83,14 +83,14 @@ format_fraction_fixed_dp <- function(x, ...) {
...
+#' Format count and fraction
 #'
 #' @description `r lifecycle::badge("stable")`
 #'
 #' Formats a count together with fraction with special consideration when count is `0`.
 #'
-#' @param x (`integer`)\cr vector of length 2, count and fraction.
-#' @param ... required for `rtables` interface.
+#' @param x (`integer`)\cr vector of length 2 with count and fraction, respectively.
+#' @param ... not used. Required for `rtables` interface.
```
Comment (Melkiades) [inline R/formatting_functions.R:93]: @martincadek this needs to be mentioned in the vignette ;)

File: R/formatting_functions.R [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
+#' Format fraction and percentage
 #'
 #' @description `r lifecycle::badge("stable")`
 #'
 #' Formats a fraction together with ratio in percent.
 #'
-#' @param x (`integer`)\cr with elements `num` and `denom`.
-#' @param ... required for `rtables` interface.
+#' @param x (named `vector` of `integer`)\cr vector with elements `num` and `denom`.
+#' @param ... not used. Required for `rtables` interface.
```
Comment (Melkiades) [inline R/formatting_functions.R:21]: reapeated isnt it? it is part of the same family?

File: R/analyze_variables.R [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 #' @description `r lifecycle::badge("stable")`
 #'
 #' Sets a list of parameters for summaries of descriptive statistics. Typically used internally to specify
 #' details for [s_summary()]. This function family is mainly used by [analyze_vars()].
 #'
 #' @inheritParams argument_convention
-#' @param quantiles (`numeric`)\cr of length two to specify the quantiles to calculate.
-#' @param quantile_type (`numeric`)\cr between 1 and 9 selecting quantile algorithms to be used.
+#' @param quantiles (`numeric`)\cr vector of length two to specify the quantiles to calculate.
+#' @param quantile_type (`numeric`)\cr number between 1 and 9 selecting quantile algorithms to be used.
```
Comment (Melkiades) [inline R/analyze_variables.R:10]: numeric(1) maybe?

File: R/analyze_variables.R [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 #'
 #' @description `r lifecycle::badge("stable")`
 #'
 #' Sets a list of parameters for summaries of descriptive statistics. Typically used internally to specify
 #' details for [s_summary()]. This function family is mainly used by [analyze_vars()].
 #'
 #' @inheritParams argument_convention
-#' @param quantiles (`numeric`)\cr of length two to specify the quantiles to calculate.
-#' @param quantile_type (`numeric`)\cr between 1 and 9 selecting quantile algorithms to be used.
+#' @param quantiles (`numeric`)\cr vector of length two to specify the quantiles to calculate.
```
Comment (Melkiades) [inline R/analyze_variables.R:9]: numeric(2) too?

File: R/abnormal_by_worst_grade_worsen.R [hunk 6]
```diff
@@ -18,16 +18,16 @@
...
 #'
-#' Helper function to prepare a `df` for generate the patient count shift table
+#' Helper function to prepare a `df` for generate the patient count shift table.
 #'
-#' @param adlb (`data.frame`)\cr `ADLB` dataframe
-#' @param worst_flag_low (named `vector`)\cr Worst low post-baseline lab grade flag variable
-#' @param worst_flag_high (named `vector`)\cr Worst high post-baseline lab grade flag variable
-#' @param direction_var (`string`)\cr Direction variable specifying the direction of the shift table of interest.
+#' @param adlb (`data.frame`)\cr ADLB data frame.
+#' @param worst_flag_low (named `vector`)\cr worst low post-baseline lab grade flag variable.
```
Comment (Melkiades) [inline R/abnormal_by_worst_grade_worsen.R:28]: character?

Comment (edelarua) [reply R/abnormal_by_worst_grade_worsen.R]: Fixed.

File: R/analyze_variables.R (hunk 5 above)
Comment (edelarua) [reply R/analyze_variables.R]: Updated.

File: R/analyze_variables.R (hunk 4 above)
Comment (edelarua) [reply R/analyze_variables.R]: Updated.

File: R/formatting_functions.R (hunk 3 above)
Comment (edelarua) [reply R/formatting_functions.R:21]: Added `@inheritParams`.

File: R/g_forest.R (hunk 1 above)
Comment (edelarua) [reply R/g_forest.R:476]: Done.

**Comment by edelarua** [comment]
@Melkiades I went through everything again and corrected/specified the parameter types better where needed.

**Review by Melkiades — APPROVED** [review]
Lgtm!!!! Thanks Emily for all your work, and for swiftly addressing my comments ;)

**Pull request merged**